homeassistant/components/fan/isy994.py

```diff
@@ -8,28 +8,33 @@
 from typing import Callable
 
 from homeassistant.components.fan import (FanEntity, DOMAIN, SPEED_OFF,
-                                          SPEED_LOW, SPEED_MEDIUM,
+                                          SPEED_LOW,
                                           SPEED_HIGH)
 import homeassistant.components.isy994 as isy
 from homeassistant.const import STATE_UNKNOWN, STATE_ON, STATE_OFF
 from homeassistant.helpers.typing import ConfigType
 
 _LOGGER = logging.getLogger(__name__)
 
+# Define term used for medium speed. This must be set as the fan component uses 
+# 'medium' which the ISY does not understand
+ISY_SPEED_MEDIUM = 'med'
+
+
 VALUE_TO_STATE = {
     0: SPEED_OFF,
     63: SPEED_LOW,
     64: SPEED_LOW,
-    190: SPEED_MEDIUM,
-    191: SPEED_MEDIUM,
+    190: ISY_SPEED_MEDIUM,
```

Comment:
No, don't change this. This should still be `SPEED_MEDIUM`. These states are the ones exposed to the user in the frontend and as attributes. Those should be the same for all fan platforms.